src/emulator.ts

```diff
@@ -151,26 +162,17 @@ export class EmacsEmulator implements IEmacsController, vscode.Disposable {
     );
 
     this.disposables = [];
+    this.commandRegistry = new EmacsCommandRegistry();
+    this.afterCommand = this.afterCommand.bind(this);
+
+    const killYanker = new KillYanker(this, killRing, minibuffer);
```

Comment:
Put these `killYanker`-related lines next to the registration of kill and yank commands as they are semantically coupled and (I think) `killYanker` is not referred to in other places.